proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/TranslatorApplication.java

```diff
@@ -90,6 +91,7 @@ private void registerResources(TranslatorConfiguration configuration, Environmen
 
     private EidasResponseGenerator createEidasResponseGenerator(TranslatorConfiguration configuration) {
         HubResponseTranslator hubResponseTranslator = new HubResponseTranslator(
+                () -> EidasResponseBuilder.instance(),
```

Comment:
In theory you could do `EidasResponseBuilder::instance,` instead of `() -> EidasResponseBuilder.instance(),`

Reply:
Applied these suggestions